pkg/crd/markers/topology.go

```diff
@@ -29,6 +29,7 @@ var TopologyMarkers = []*definitionWithHelp{
 	must(markers.MakeDefinition("listType", markers.DescribesField, ListType(""))),
 	must(markers.MakeDefinition("listMapKey", markers.DescribesField, ListMapKey(""))),
 	must(markers.MakeDefinition("mapType", markers.DescribesField, MapType(""))),
+	must(markers.MakeDefinition("structType", markers.DescribesField, StructType(""))),
```

Comment:
Keeping them ordered, sweet

Reply:
listType and listMapKey are not ordered :-/ Not your fault ...

Reply:
> listType and listMapKey are not ordered

Shall I change that as part of this PR? It's a tiny bit of scope creep, but especially if we'll squash commits as part of https://github.com/kubernetes-sigs/controller-tools/pull/347, it's probably not a huge deal.

Reply:
Yeah, I'll re-order when I rebase/squash! :-)